xml/System.Drawing/Graphics.xml

```diff
@@ -1703,7 +1703,7 @@
   
 -   Creates the start, end, and two control points for a first curve and endpoint and two control points for a second curve.  
   
--   Draws the successive Bézier curves to the screen.  
+-   Draws the successive B�zier curves to the screen.  
```

Comment:
This should probably be reverted (and fixed in the tool).

Reply:
Is it correct that all the xml files in the docs repo should be encoded with code page 1252 ? the tool was reading them this way, but not writing. With my recent change, it's writing too. It may be that these files were before that change.

Reply:
I'm not sure, I'll check.